### DISCUSSION TITLE
Border around selected/hovered items

### DISCUSSION THREAD

**Discussion by ACR-Jeff** [jellyfin/jellyfin-androidtv#4035] [General]
I have setup a jellyfin server for my elderly parents, Some times it's difficult for them to see what item they have selected/hovered over, Is there possibly a way to achieve a border around the selected items such as cards, Maybe CSS? Thank you

**Comment by nielsvanvelzen** [comment]
There is no such thing as CSS on the Android platform. Improving the UI of the app is an ongoing effort, marking the focused item is one of the things that will be changed eventually.


I'll most likely copy over the effect I used in another app I work on:

[957000040b.webm](https://github.com/user-attachments/assets/ac0eb3d7-30e1-4b57-8df7-3ab2e39b7ace)

This uses a combination of scale and a border with the color based on the image it shows.

**Comment by ACR-Jeff** [comment]
Thank you very much. I look forward to this update.
